app/helpers/schemes_helper.rb

```diff
@@ -44,6 +44,14 @@ def toggle_scheme_link(scheme)
     return govuk_button_link_to "Reactivate this scheme", scheme_new_reactivation_path(scheme) if scheme.deactivated?
   end
 
+  def edit_scheme_text(scheme, user)
+    if user.data_provider?
+      "If you think this scheme should be updated, ask a data coordinator to make the changes. Find your data coordinators on the #{link_to('users page', users_path)}.".html_safe
+    elsif user.data_coordinator? && user.organisation.parent_organisations.include?(scheme.owning_organisation)
+      "This scheme belongs to your stock owner #{scheme.owning_organisation.name}."
```

Comment:
should there be a comma here after owner?

Reply:
Not based on the designs